pallets/utxo/src/lib.rs

```diff
@@ -267,7 +267,9 @@ pub mod pallet {
         CreatePP(Vec<u8>, Vec<u8>),
         /// Pay to an existing contract. Takes a destination account,
         /// whether the call funds the contract, and input data.
```

Comment:
I think the comment here needs to be updated `CallPP` no longer takes the boolean of whether the call funds the contract.